projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.ts

```diff
@@ -19,7 +20,8 @@ export class BreadcrumbComponent implements OnInit {
 
   constructor(
     public component: CmsComponentData<CmsBreadcrumbsComponent>,
-    protected pageMetaService: PageMetaService
+    protected pageMetaService: PageMetaService,
```

Comment:
Probably You need to update the providers in the spec file of the breadcrumb component as well

Reply:
It turned out tha test suite for breadcrumbs is skipped for more than year now. Let me update the spec file.